### PR TITLE
Fix typo and intendation in aggregator-workflow

### PR DESCRIPTION
**Pull request by martenole**
Ping @chiarazampolli

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Thanks! As soon as it is merged, please update O2 on the EPNs.

File: prodtests/full-system-test/aggregator-workflow.sh [hunk 1]
```diff
@@ -99,23 +99,22 @@ if workflow_has_parameter CALIB_PROXIES; then
       add_W o2-dpl-raw-proxy "--dataspec \"$CALIBDATASPEC_BARREL_SPORADIC\" $(get_proxy_connection barrel_sp input)" "" 0
     fi
   elif [[ $AGGREGATOR_TASKS == TPCIDC_BOTH ]]; then
-    if [[ ! -z $CALIBDATASPEC_TPCIDC_A ]] || [[ ! -z $CALIBDATASPEC_TPCIDC_C ]]; then
+    if [[ ! -z $CALIBDATASPEC_TPCIDC_A ]] && [[ ! -z $CALIBDATASPEC_TPCIDC_C ]]; then
```
Comment (martenole) [inline prodtests/full-system-test/aggregator-workflow.sh]: I think here we need to make sure that both variables are set, right?

File: prodtests/full-system-test/aggregator-workflow.sh [hunk 2]
```diff
@@ -99,23 +99,22 @@ if workflow_has_parameter CALIB_PROXIES; then
...
+          exit 2
+        fi
+        # define port for FLP; should be in 47900 - 47999; if nobody defined it, we use 47900
+        [[ -z $TPC_IDC_FLP_PORT ]] && TPC_IDC_FLP_PORT=47900
+        # expand FLPs; TPC uses from 001 to 145, but 145 is reserved for SAC
+        for flp in $(seq -f "%03g" 1 144); do
+          FLP_ADDRESS="tcp://alicr1-flp-ib${flp}:${TPC_IDC_FLP_PORT}"
+          CHANNELS_LIST+=" --channel-config \"type=pull,name=tpcidc_flp${flp},transport=zmq,address=$FLP_ADDRESS,method=connect,rateLogging=10\""
+        done
+        add_W o2-dpl-raw-proxy "--dataspec \"$CALIBDATASPEC_TPCIDC_A;$CALIBDATASPEC_TPCIDC_C\" $CHANNELS_LIST --timeframes-shm-limit $TIMEFRAME_SHM_LIMIT" "" 0
```
Comment (martenole) [inline prodtests/full-system-test/aggregator-workflow.sh:115]: And here there was one $ missing in front of `CALIBDATASPEC_TPCIDC_C`

File: prodtests/full-system-test/aggregator-workflow.sh (hunk 1 above)
Comment (chiarazampolli) [reply prodtests/full-system-test/aggregator-workflow.sh]: No, i wanted an OR explicitely: if at least one is there.

File: prodtests/full-system-test/aggregator-workflow.sh (hunk 2 above)
Comment (chiarazampolli) [reply prodtests/full-system-test/aggregator-workflow.sh:115]: Yes!!! :-(

File: prodtests/full-system-test/aggregator-workflow.sh (hunk 1 above)
Comment (martenole) [reply prodtests/full-system-test/aggregator-workflow.sh]: Ok, understood. I changed it back

**Comment by martenole** [comment]
Force-merging now, since this is needed for P2 operations

**Pull request merged**